torchqdynamics/sesolve/propagator.py

```diff
@@ -8,11 +8,9 @@ class SEPropagator(Solver):
     def __init__(self, *args):
         super().__init__(*args)
 
-        self.H = self.H[:, None, ...]  # (b_H, 1, n, n)
-
     def run(self):
         y, t1 = self.y0, 0.0
         for t2 in tqdm(self.t_save, disable=not self.options.verbose):
-            y = torch.matrix_exp(-1j * self.H * (t2 - t1)) @ y
+            y = torch.matrix_exp(-1j * self.H(t2) * (t2 - t1)) @ y
```

Comment:
A priori the `propogator` method is always time independent. We will have to check this somewhere I guess, but it can be a separate PR.